### ISSUE TITLE
Add support for importing ES mappings from CSV files

### ISSUE THREAD

**Issue by alpgarcia** [grimoirelab/panels#117]
Definition of enriched indexes are stored in [schema folder](https://github.com/grimoirelab/panels/tree/master/schema). We need to use them to compare those definitions against ElasticSearch mapping generated by Gelk, that is, real mappings.

In order to do that we need to add a `from_csv()` method that works in the same way as `from_json()` does in [ESMapping class](https://github.com/grimoirelab/panels/blob/45529b660631871d9c133eadf920f32bb372180d/src/owlwatch/schema/model.py#L232). It should load a CSV file and use the `add_property` method to create the Index Pattern.

**Comment by alpgarcia** [comment]
I'm currently working on this. I found some issues in the process:
* CSV files needs a way to define when `text` fields should be aggregatable. By default all fields are aggregatable by default except `text` fields. I added a workaround considering that in our current CSVs, `text` implies being `aggregatable`. **This should be removed when CSVs are improved.**
* CSV files are not specifying index names. As a workaround, I used file name without file extension for this.

Current staus: method `from_csv()` added to `ESMapping` class in `model.py`. Owlwatch command line tool is in process of supporting it. Support for comparing CSVs vs Panels is done. Support for comparing CSVs to mappings from ES is WIP.
